resources/views/auth/register.blade.php

```diff
@@ -162,6 +174,15 @@ class="mt-2"
             />
         @endif
 
+        @if ($githubUsername)
+            <x-text-input
```

Comment:
We add a hidden field to be able to send back the GitHub username back to the registration controller